docker/Dockerfile.windows.1909

```diff
@@ -4,8 +4,19 @@ FROM mcr.microsoft.com/windows/servercore:1909 AS git
 SHELL ["powershell.exe", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; `
-    Invoke-WebRequest -UseBasicParsing https://github.com/git-for-windows/git/releases/download/v2.21.0.windows.1/MinGit-2.21.0-64-bit.zip -OutFile git.zip; `
-    Expand-Archive git.zip -DestinationPath C:\git;
+    Invoke-WebRequest -UseBasicParsing https://github.com/activescott/lessmsi/releases/download/v1.6.91/lessmsi-v1.6.91.zip -OutFile $env:TEMP\lessmsi.zip; `
+    Expand-Archive $env:TEMP\lessmsi.zip -DestinationPath $env:TEMP;
+
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; `
+    Invoke-WebRequest -UseBasicParsing https://www.7-zip.org/a/7z1604-x64.msi -OutFile $env:TEMP\7z.msi; `
+    cd $env:TEMP; `
```

Comment:
Extract everything to temp dir, so image stays small.

Reply:
Same here, comment would be helpful.